frontend/app/src/components/modals/CreateOven.tsx

```diff
@@ -44,7 +44,7 @@ interface IDepositorItem {
 
 interface ICreateVaultForm {
   delegate: string;
-  amount: number;
+  amount: number | '';
```

Comment:
This idea is good, we can implement this for all forms

Reply:
yeah, i have implemented this for all the input fields